deployment/kubernetes/mcp-servers/math-mcp/server.py

```
@@ -0,0 +1,261 @@
+
+        elif operation == "divide":
+            if op.a is None or op.b is None:
+                raise HTTPException(status_code=400, detail="Parameters 'a' and 'b' are required")
+            if op.b == 0:
+                raise HTTPException(status_code=400, detail="Division by zero is not allowed")
+            result = op.a / op.b
+            return MathResult(result=result, message=f"{op.a} ÷ {op.b} = {result}")
+
+        elif operation == "power":
+            if op.base is None or op.exponent is None:
+                raise HTTPException(status_code=400, detail="Parameters 'base' and 'exponent' are required")
```

Ruff (0.15.2)

[warning] Abstract `raise` to an inner function

(TRY301)